### DISCUSSION TITLE
Add "location" as a field to service calls to Google and Local Calendars

### DISCUSSION THREAD

**Discussion by luca-angemi** [home-assistant/architecture#885] [Entity Models] [answered]
## Context
Currently Google and Local calendars don't have the option to add a location through the service call "Create Event"

## Proposal
Add the new field to the correspondant service calls in line with the rfc
https://www.rfc-editor.org/rfc/rfc5545#section-3.8.1.7. 
I've opened a PR: https://github.com/home-assistant/core/pull/90098

## Consequences
The user can add the location of an event through service calls.

**Comment by MartinHjelmare** [comment, marked as answer]
Sounds good.
